src/module/api/update/index.ts

```diff
@@ -0,0 +1,5 @@
+import * as search from "./search";
+
+export const Update = {
+  Search: search.Search,
+};
```

Comment:
The update service wrapping functionality around the Update resource implements a Search function. This can be used to search for updates.